### PR TITLE
adding image to demo yaml with demo image policy

### PR DESCRIPTION
**Pull request by larslunde-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/fees-pay/dead-letter-queue-process/demo-00.yaml [hunk 1]
```diff
@@ -5,5 +5,6 @@ metadata:
 spec:
   values:
     job:
+      image: "hmctspublic.azurecr.io/payment/jobs:prod-0a0ded6-20210716134918"   #{"$imagepolicy": "flux-system:demo-dead-letter-queue-process"}
```
Comment (thomast1906) [inline k8s/namespaces/fees-pay/dead-letter-queue-process/demo-00.yaml:8]: This imagepolicy is going to pull a PR image: pr-30-* - do you need to update this?

**Comment by thomast1906** [comment]
Also, can you kustomize correctly with using https://github.com/hmcts/cnp-flux-config/tree/master/k8s/namespaces/fees-pay/dead-letter-queue-process etc - rather than directly only cluster?

File: k8s/namespaces/fees-pay/dead-letter-queue-process/demo-00.yaml (hunk 1 above)
Comment (larslunde-hmcts) [reply k8s/namespaces/fees-pay/dead-letter-queue-process/demo-00.yaml:8]: Yeah that is what we want, working with Hannah Smith on this to get this PR correct

**Comment by larslunde-hmcts** [comment]
Closing to start again, re-named other env files on accident.

**Pull request closed without merging**